pyproject.toml

```
@@ -23,11 +23,11 @@ classifiers = [
     "Intended Audience :: Developers",
 ]
 dependencies = [
-    "ghga-service-commons[all]>=0.7.1,<0.8",
-    "hexkit>=0.10.2,<0.11",
-    "crypt4gh>=1.6,<2",
-    "hvac>=2,<3",
-    "pydantic>=1.10.6,<2"
+    "ghga-service-commons[all]>=1.0.1",
```

```suggestion
    "ghga-service-commons[all]>=1.1.0",
```
If we want the most recent one as lower bound

---

Yes, why not :)